pytensor/tensor/pad.py

```diff
@@ -553,7 +553,7 @@ def pad(x: TensorLike, pad_width: TensorLike, mode: PadMode = "constant", **kwar
 
         print(pt.pad(a, (2, 3), 'linear_ramp', end_values=(5, -4)).eval())
 
-    ..testoutput::
+    .. testoutput::
```

Comment:
Obviously. Sphinx  / rst is the worst

Reply:
Seems like we could have a pre-commit hook for that